### DISCUSSION TITLE
How to export a note to MD without links to pdf in highlights

### DISCUSSION THREAD

**Discussion by aulline13** [windingwind/zotero-better-notes#848] [Q&A]
Currently, when I export a note to MD using Zotero Better Notes, I get the following result in Obsidian:

![image](https://github.com/windingwind/zotero-better-notes/assets/99823829/3c0d5bb6-e970-48e3-b493-309e4d2d1832)

Both the highlight (first line on the picture) and the citation (second line) are linked to Zotero pdf. I would like to see the link only for the citation, i.e. it should look like this:

![image](https://github.com/windingwind/zotero-better-notes/assets/99823829/574bdd54-9b17-48e7-a238-a05d6bf37c73)

Maybe it is easily configurable, but I can't find a way to change the behaviour on my own. Sorry if it's a dumb question.

P.S. When I export to MD using Zotero's native export feature, I get the desired result straight away, with only citations, not highlights, linking to pdf.

**Comment by koenraijer** [comment]
I second this! Was just about to ask the same question.
Though I will also use this opportunity to compliment @windingwind on the amazing and free functionality. You're doing us all a gigantic service.

I will add to the @aulline13's post by posting the desired code:

The current behaviour:
```
> <span class="highlight" data-annotation="%7B%22attachmentURI%22%3A%22http%3A%2F%2Fzotero.org%2Fusers%2F9323887%2Fitems%2FR83WEHNQ%22%2C%22annotationKey%22%3A%22BT9MFB25%22%2C%22color%22%3A%22%23ffd400%22%2C%22pageLabel%22%3A%222%22%2C%22position%22%3A%7B%22pageIndex%22%3A1%2C%22rects%22%3A%5B%5B256.96%2C307.47%2C539.858%2C316.27%5D%2C%5B246.92%2C295.65%2C447.205%2C305.05%5D%5D%7D%2C%22citationItem%22%3A%7B%22uris%22%3A%5B%22http%3A%2F%2Fzotero.org%2Fusers%2F9323887%2Fitems%2FKRHQTBVR%22%5D%2C%22locator%22%3A%222%22%7D%7D" ztype="zhighlight"><a href="zotero://open-pdf/library/items/R83WEHNQ?page=2&#x26;annotation=BT9MFB25">Like a river whose eddies, vortices, and turbulent structures do not exist independent of the flow itself, so it is with the brain.</a></span> <span class="citation" data-citation="%7B%22citationItems%22%3A%5B%7B%22uris%22%3A%5B%22http%3A%2F%2Fzotero.org%2Fusers%2F9323887%2Fitems%2FKRHQTBVR%22%5D%7D%5D%2C%22properties%22%3A%7B%7D%7D" ztype="zcitation">(<span class="citation-item"><a href="zotero://select/library/items/KRHQTBVR">Kelso, 1995</a></span>)</span>
```

The desired behaviour:
```
> Like a river whose eddies, vortices, and turbulent structures do not exist independent of the flow itself, so it is with the brain. ([Kelso, 1995](zotero://select/library/items/KRHQTBVR)) ([pdf](zotero://open-pdf/library/items/R83WEHNQ?page=2&annotation=BT9MFB25))
```

In a similar vein, I really enjoy that images are finally able to be exported with the note, but instead of a simple image tag, they show up like this:
```
![\<img alt="" data-attachment-key="I8Z8QE3T" data-annotation="%7B%22attachmentURI%22%3A%22http%3A%2F%2Fzotero.org%2Fusers%2F9323887%2Fitems%2FR83WEHNQ%22%2C%22annotationKey%22%3A%22GH644XYI%22%2C%22color%22%3A%22%23ffd400%22%2C%22pageLabel%22%3A%2210%22%2C%22position%22%3A%7B%22pageIndex%22%3A9%2C%22rects%22%3A%5B%5B251.786%2C537.536%2C407.679%2C716.464%5D%5D%7D%2C%22citationItem%22%3A%7B%22uris%22%3A%5B%22http%3A%2F%2Fzotero.org%2Fusers%2F9323887%2Fitems%2FKRHQTBVR%22%5D%2C%22locator%22%3A%2210%22%7D%7D" width="260" height="298" src="attachments/I8Z8QE3T.png" ztype="zimage">](attachments/I8Z8QE3T.png)
```

Whereas what I would like is:
```
![[notes/attachments/I8Z8QE3T.png]]
```

As a bonus of something I'd like to understand, is it possible to change the folder where the attachments end up? E.g., I want them to be in "_assets/", which is a sibling of the folder the note is in. 

Would be great if anyone was able to help me/us out!

**Reply by ellinum** [reply]
This would indeed be such a welcome addition. 

With the whole highlight-text linked, it is really hard on the eyes in Obsidian AND it makes the text hard to modify, because when you move the cursor into the, you start seeing the whole link markdowncode.

**Comment by aidecameron** [comment]
You can try my template AF2N, which implements an easier handlebar template language within BN template. You can redefine things without any typescripts. Please see [https://github.com/aidecameron/zotero-af2n](https://github.com/aidecameron/zotero-af2n)
